src/coreclr/jit/importer.cpp

```diff
@@ -8813,8 +8829,9 @@ void Compiler::impImportBlockCode(BasicBlock* block)
 
                 JITDUMP(" %08X", resolvedToken.token);
 
-                int      aflags = isLoadAddress ? CORINFO_ACCESS_ADDRESS : CORINFO_ACCESS_GET;
-                GenTree* obj    = nullptr;
+                GenTreeFlags indirFlags = impPrefixFlagsToIndirFlags(prefixFlags);
```

Comment:
These aren't used until ~170 lines later, can we move this down closer to the use?

Reply:
In the store case (below) there are quite a few more uses, so if you want to leave things this way for symmetry then perhaps mark it as const so it is clearer  nothing in between is modifying it?

Reply:
The flags may get modified by the call `impImportStaticFieldAddress`, this happens when importing a static readonly `object` field. I can move both of these declarations closer to their uses if you think it would be better (the motivation for having this them this high up was indeed symmetry).